reference.md

```diff
@@ -1255,14 +1274,18 @@ can use `tostring` to get the name of a symbol.
 
 * `list` - return a list, which is a special kind of table used for code
 * `sym` - turn a string into a symbol
-* `list?` - is the argument a list?
-* `sym?` - is the argument a symbol?
-* `table?` - is the argument a non-list table?
+* `gensym` - generates a unique symbol for use in macros (also available via
+             `<var-name>#`).
+* `list?` - is the argument a list? Returns `argument` or `false`.
```

Comment:
Maybe this is a bit clearer?

```suggestion
* `list?` - is the argument a list? Returns its argument or `false`.
```

Reply:
_the_ argument? That follows the previous sentence.

- `list?` is the argument a list? Returns the argument or `false`.